supertokens_python/recipe/emailpassword/api/signup.py

```diff
@@ -15,6 +15,14 @@
 
 from typing import TYPE_CHECKING, Any
```

Comment:
please also change the patch version and changelog. + link to issue in changelog and in PR description